doc/docs/tutorials/single_machine.md

```diff
@@ -88,38 +146,104 @@ ERROR o.openmrs.analytics.PipelineManager o.openmrs.analytics.PipelineManager$Pi
 pipeline-controller    | java.sql.SQLException: org.apache.hive.service.cli.HiveSQLException: Error running query: org.apache.spark.sql.AnalysisException: Unable to infer schema for Parquet. It must be specified manually.
 ```
 
-try running `sudo chmod -R 755` on the Parquet file directory, by default located at `docker/dwh`.
-
-## View and analyze the data using Spark Thrift server
-
-Connect to the Spark Thrift server using a client that supports Apache Hive. For example, if using the JDBC driver, the URL should be `jdbc:hive2://localhost:10001`. The pipeline will automatically create `Patient`, `Encounter`, and `Observation` tables when run.
-
-Let's do some basic quality checks to make sure the data is uploaded properly (note
-table names are case in-sensitive):
-
-```hiveql
-SELECT COUNT(0) FROM Patient;
-```
-We should have exactly 79 patients:
-```
-+-----------+
-| count(0)  |
-+-----------+
-| 79        |
-+-----------+
-```
-
-Doing the same for observations:
-```hiveql
-SELECT COUNT(0) FROM Observation;
-```
-```
-+-----------+
-| count(0)  |
-+-----------+
-| 17279     |
-+-----------+
-```
-
-## What's next
-Now that the data is available in an SQL queryable format, you can start to explore it using SQL or jupyter notebooks or build dashboards using common open source tools like Apache SuperSet.
\ No newline at end of file
+Try running `sudo chmod -R 755` on the Parquet file directory, by default
+located at `docker/dwh`.
+
+## Explore the resulting schema in PostgreSQL
+
+If you set the `sinkDbConfigPath` parameter to output views to a PostgreSQL DWH,
+connect to the DB via docker using this
+cmd: `docker exec -it <container_name_or_id> bash`
+
+If using the default container (hapi-fhir-db)
+run: `docker exec -it hapi_fhir_db bash`
+
+Using psql connect to the 'views' database: `psql -U admin -d views`
+
+To list the tables: `\d`. It should look something like this:
+
+| Schema | Name                                                                                                                            | Type  | Owner | 
+|--------|---------------------------------------------------------------------------------------------------------------------------------|-------|-------|
+| public | [condition_flat](https://github.com/google/fhir-data-pipes/blob/master/docker/config/views/Condition_flat.sql)                  | table | admin |
```

Comment:
These are links to the SQL versions of the views, which is for Parquet+Spark. We should instead link the ViewDefinition versions, e.g., [condition_flat_view](https://github.com/google/fhir-data-pipes/blob/master/docker/config/views/condition_flat_view.json).